rasa/utils/tensorflow/environment.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import os
+from typing import Text
+
+import tensorflow as tf
```

Comment:
Should I just do `from tensorflow import config as tf_config` globally? That should take care of warnings as well as it would only import the `tf.config` submodule?

Reply:
Pycharm still complains if I do that.

Reply:
Not sure why

Reply:
it'll still import the whole tensorflow
let's use `tf.config`

Reply:
what version of tf your PyCharm uses?

Reply:
No import it in the functions please

Reply:
Ok, removed the global import, added `from tensorflow import config as tf_config` in individual functions.